docs_src/_layouts/home.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+{% include head.html %}
+
+
+<body>
+
+{% include header.html %}
+
+
+<div id="home" class="container">
+
+
+
+    <div class="summary row">
+        <div class="col-lg-6 pt-4 pt-lg-0 order-2 order-lg-1 d-flex flex-column justify-content-center footer-top">
+            <h1>We are DevX!</h1>
+            <h2>We study how humans interact with code and design new ways to build software. I work at the intersection
+                of software engineering and human-computer interaction.</h2>
+        </div>
+        <div class="col-lg-6 order-1 order-lg-2 hero-img">
+            <img src="{{ site.baseurl }}/assets/img/gallery/oh-george-summer19.jpg" class="img-fluid" alt="Group Photo">
+        </div>
```

Comment:
this could be obtained from the md file

Reply:
Only the image or the whole content? How?